app/screens/TalkDetailsScreen/TalkDetailsScreen.tsx

```diff
@@ -181,14 +178,17 @@ const TalkDetailsSingleSpeaker: React.FunctionComponent<TalkDetailsSingleSpeaker
   }) {
     const hasSocialButtons = socialButtons.some((button) => button.url)
     const offset = 6
+    const { width: screenWidth } = useWindowDimensions()
+    const screenContentWidth = screenWidth - layout.horizontalGutter * 2
```

Comment:
I would love to extract this computation out, but not sure what I want it to be yet.